### PR TITLE
PRM-584 Added Additional Penalty Rate

### PR DESCRIPTION
**Pull request by ruthdas03**
Screenshot 2021-09-06 at 15 59 37

### CODE REVIEW

File: app/controllers/CalculationController.scala [hunk 1]
```diff
@@ -65,7 +65,8 @@ class CalculationController @Inject()(viewLPP: CalculationLPPView, viewAdd: Calc
               }
             )
           } else {
-            Ok(viewAdd())
+            val additionalPenaltyRate = parseBigDecimalToFriendlyValue(penalty.get.financial.percentageOfOutstandingAmtCharged.getOrElse(0))
```
Comment (LewisMitchell) [inline app/controllers/CalculationController.scala:68]: Not sure if we should default the additional penalty rate - to be honest, I'm also not sure if we should use the payload as we know it's going to be 4%?
Whichever way you do it, I'd say if the value was None then we should show an error.

File: it/controllers/CalculationControllerISpec.scala [hunk 2]
```diff
@@ -236,7 +236,7 @@ class CalculationControllerISpec extends IntegrationSpecCommonBase {
       parsedBody.select("#main-content tr").get(1).select("th").text() shouldBe "Number of days since day 31"
       parsedBody.select("#main-content tr").get(1).select("td").text() shouldBe "0 days" //TODO: placeholder value
       parsedBody.select("#main-content tr").get(2).select("th").text() shouldBe "Additional penalty rate"
-      parsedBody.select("#main-content tr").get(2).select("td").text() shouldBe "0%" //TODO: placeholder value
+      parsedBody.select("#main-content tr").get(2).select("td").text() shouldBe "2%"
```
Comment (LewisMitchell) [inline it/controllers/CalculationControllerISpec.scala:239]: As its additional - could you make this 4%?

File: test/views/CalculationViewSpec.scala [hunk 3]
```diff
@@ -58,7 +58,7 @@ class CalculationViewSpec extends SpecBase with ViewBehaviours with ViewUtils {
         Selector.listRow(2) -> th2Additional,
         //    Selector.listValue(2) -> "0% of £0 (VAT amount unpaid on 0)" //TODO: Implement with actual values
         Selector.listRow(3) -> th3Additional,
-        //    Selector.listValue(3) -> "£0" //TODO: Implement with actual values
+        Selector.listValue(3) -> "2%",
```
Comment (LewisMitchell) [inline test/views/CalculationViewSpec.scala:61]: As above - could you make this 4%?

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
Looks good - just a query and some changes 😄

File: app/controllers/CalculationController.scala (hunk 1 above)
Comment (ruthdas03) [reply app/controllers/CalculationController.scala:68]: Ok, in that case adding the 'additional penalty rate' as a static value of 4

File: test/views/CalculationViewSpec.scala (hunk 3 above)
Comment (ruthdas03) [reply test/views/CalculationViewSpec.scala]: Done

**Review by LewisMitchell — APPROVED** [review]

**Pull request merged**